samples/msgext-ai-sentiment-analysis/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-msgext-ai-sentiment-analysis-nodejs",
+    "source": "officeDev",
+    "title": "Sentiment analysis with messaging extension and Azure Open AI",
+    "shortDescription": "Sentiment analysis Using Azure Open AI through messaging extension.",
```

Comment:
Sentiment analysis of Teams chat message using messaging extension and Azure Open AI

Reply:
fixed